server.js

```diff
@@ -56,7 +56,7 @@ app.prepare().then(() => {
 
   // DO NOT REMOVE
   server.get('/api', (request, res) => {
-    res.redirect('https://geo.api.gouv.fr/adresse')
+    res.redirect('api-doc')
```

Comment:
"Any file inside the folder pages/api is mapped to /api/* and will be treated as an API endpoint instead of a page."

[source](https://nextjs.org/docs/api-routes/introduction)